app/src/main/java/org/jellyfin/androidtv/ui/playback/CustomPlaybackOverlayFragment.java

```diff
@@ -1487,4 +1494,29 @@ private void renderSubtitles(@Nullable final String text) {
             binding.subtitlesText.setVisibility(View.VISIBLE);
         });
     }
+
+    public boolean getServerMeetsMinimumVersion(@NonNull String minServerVersion) {
```

Comment:
```suggestion
    public boolean getServerMeetsMinimumVersion(@NonNull ServerVersion minServerVersion) {
```
Use an instance of server version instead of a string, the constructor is easy to use. Should look something like `new ServerVersion(10, 8, 0);` from Java.